### PR TITLE
[NeedsReview] b07902001

### PR DESCRIPTION
**Pull request by jameshwc**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is there any suggestion?

<br />

### CODE REVIEW

**Review by amjltc295 — CHANGES_REQUESTED** [review]
Please include only your studentID.py in this PR. Also, it would be better to revise your commit message.

File: src/students/b07902001.py [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+
+    Returns:
+        result_img: an PIL Image
+
+    Hints:
+        * Make sure you have installed the PIL package
+        * Take a look at utils.py first
+        * You could easily find answers with Google
+    '''
+    # TODO: download the image from img_url with the request module
```
Comment (amjltc295) [inline src/students/b07902001.py]: Did you give up the task 8?

**Review by amjltc295 — APPROVED** [review]
You could choose not to change your code.

File: src/students/b07902001.py (hunk 1 above)
Comment (jameshwc) [reply src/students/b07902001.py]: I have done it. Please check the Line 380 to Line 396. Thanks.

File: src/students/b07902001.py [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+    return student
+
+
+def task_8(
+    img_url: str = 'https://i.imgur.com/B75zq0x.jpg'
+) -> object:
+    from urllib import request
+    from PIL import Image, ImageFont, ImageDraw
+
+    def draw_text(
```
Comment (amjltc295) [inline src/students/b07902001.py]: Please do not define draw_text() again as it is defined in utils.py. Import it and use it.

**Review by amjltc295 — APPROVED** [review]

File: src/students/b07902001.py (hunk 1 above)
Comment (amjltc295) [reply src/students/b07902001.py]: @jameshwc okay, reviewed.

File: src/students/b07902001.py (hunk 2 above)
Comment (jameshwc) [reply src/students/b07902001.py]: Done it. Please review. Thx.

**Pull request merged**